public/3D.js

```diff
@@ -94,19 +89,10 @@ function init3D() {
 	allObjects.push(table);
 	allObjects.push(plant);
 
-	document.getElementById("open").hidden = false;
-
-/*
-	//choose which object to make when the makeobjectfunction is called
-	geometry = new THREE.BoxGeometry(10, 20, 10);
-	material = new THREE.MeshBasicMaterial( {color: 0x669966, wireframe: false});
-	object = new THREE.Mesh(geometry, material);
-	allObjects.push(object);
-*/
-	
+	document.getElementById("open").hidden = false;	
```

Comment:
Some of these HTML elements have been renamed, but hopefully that is fixed in the merge.